src/index.html

```diff
@@ -6,9 +6,48 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__image">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
```

Comment:
Small style issue, better have spaces between **multiline** siblings, (e.g between these `<li>` because they take 3 lines).
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>

          <li class="nav__item">
            <a href="#smartphones" class="nav__link">Smartphones</a>
          </li>

```